### PR TITLE
[Android] Implement API for updating pin notes

### PR DESCRIPTION
**Pull request by tyaporush**
## Description for the changelog
* Pin/Unpin note support for remote api

## Checklist

- [x] Build and linting is passing.
- [x] This change is not breaking existing flow of a system.
- [x] I have written test case for this change.
- [x] This change is tested from all aspects.
- [ ] Implemented any new third-party library _(Which not existed before this change)_.

### CODE REVIEW

File: noty-android/data/remote/src/main/java/dev/shreyaspatil/noty/data/remote/api/NotyService.kt [hunk 1]
```diff
@@ -17,15 +17,11 @@
...
 import dev.shreyaspatil.noty.data.remote.model.response.NoteResponse
 import dev.shreyaspatil.noty.data.remote.model.response.NotesResponse
 import retrofit2.Response
-import retrofit2.http.Body
-import retrofit2.http.DELETE
-import retrofit2.http.GET
-import retrofit2.http.POST
-import retrofit2.http.PUT
-import retrofit2.http.Path
+import retrofit2.http.*
```
Comment (PatilShreyas) [inline noty-android/data/remote/src/main/java/dev/shreyaspatil/noty/data/remote/api/NotyService.kt:24]: @tyaporush can you please revert this change? It's better to have no-wildcard imports

**Review by PatilShreyas — COMMENTED** [review]
@tyaporush Thanks for raising PR 😄. Just one small comment and I would be happy to merge it once CI is passed.

**Comment by PatilShreyas** [comment]
@tyaporush CI is failing, can you look into it?

File: noty-android/data/remote/src/main/java/dev/shreyaspatil/noty/data/remote/api/NotyService.kt (hunk 1 above)
Comment (tyaporush) [reply noty-android/data/remote/src/main/java/dev/shreyaspatil/noty/data/remote/api/NotyService.kt]: @PatilShreyas Removed Wildcard imports

**Review by PatilShreyas — APPROVED** [review]
Thanks, @tyaporush, it's a very quick one. Change looks good to me 👍🏻

**Comment by tyaporush** [comment]
> Thanks, @tyaporush, it's a very quick one. Change looks good to me 👍🏻

Thanks @PatilShreyas Would love to contribute more 😊
